### PR TITLE
[7.0.0] Add rocky linux docker image for IS 7.0

### PR DESCRIPTION
**Pull request by ZiyamSanthosh**
$subject

Related issue:
- https://github.com/wso2-enterprise/wso2-iam-internal/issues/1769

### CODE REVIEW

File: dockerfiles/jdk17/rocky/is/docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+# ------------------------------------------------------------------------
+# Copyright 2022 WSO2, LLC. (http://wso2.com)
```
Comment (chamathns) [inline dockerfiles/jdk17/rocky/is/docker-entrypoint.sh:3]: update license header

File: dockerfiles/rocky/is/docker-entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+# ------------------------------------------------------------------------
+# Copyright 2022 WSO2, LLC. (http://wso2.com)
```
Comment (chamathns) [inline dockerfiles/rocky/is/docker-entrypoint.sh:3]: update license header

File: dockerfiles/jdk17/rocky/is/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+ENV JAVA_VERSION jdk-17.0.9+9
+
+# Install JDK17
+RUN set -eux; \
+    ARCH="$(objdump="$(command -v objdump)" && objdump --file-headers "$objdump" | awk -F '[:,]+[[:space:]]+' '$1 == "architecture" { print $2 }')"; \
+    case "${ARCH}" in \
+        amd64|i386:x86-64) \
+            ESUM='7b175dbe0d6e3c9c23b6ed96449b018308d8fc94a5ecd9c0df8b8bc376c3c18a'; \
+            BINARY_URL='https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.9%2B9/OpenJDK17U-jdk_x64_linux_hotspot_17.0.9_9.tar.gz'; \
```
Comment (chamathns) [inline dockerfiles/jdk17/rocky/is/Dockerfile:38]: should be 17.0.10
`17.0.10_7`

Comment (ZiyamSanthosh) [reply dockerfiles/jdk17/rocky/is/Dockerfile:38]: Added jdk-17.0.9 with arm64 support

File: dockerfiles/jdk17/rocky/is/docker-entrypoint.sh (hunk 1 above)
Comment (ZiyamSanthosh) [reply dockerfiles/jdk17/rocky/is/docker-entrypoint.sh]: Addressed with https://github.com/wso2/docker-is/pull/418/commits/0db06b1a24e55a428dcdc4f6621fdf1fc9f314fc

File: dockerfiles/rocky/is/docker-entrypoint.sh (hunk 2 above)
Comment (ZiyamSanthosh) [reply dockerfiles/rocky/is/docker-entrypoint.sh]: Addressed with https://github.com/wso2/docker-is/pull/418/commits/0db06b1a24e55a428dcdc4f6621fdf1fc9f314fc

**Review by chamathns — APPROVED** [review]

**Pull request merged**